pkg/virt-handler/vm.go

```diff
@@ -286,21 +286,22 @@ func handleDomainNotifyPipe(domainPipeStopChan chan struct{}, ln net.Listener, v
 	// Listen for new connections,
 	// Close listener and exit when stop encountered
 	go func(vmi *v1.VirtualMachineInstance, ln net.Listener, domainPipeStopChan chan struct{}) {
+		go func() {
+			<-domainPipeStopChan
+			log.Log.Object(vmi).Infof("closing notify pipe listener for vmi")
+			ln.Close()
```

Comment:
i just noticed that Close() can return an err. is that something we need to handle?

Reply:
We could at least log an error when it happens, but I don't think there is much else we can do.
It would probably error if the underlying socket is already closed or does not exist anymore. But in our case we should be fine either way?